raft/src/main/java/org/apache/kafka/raft/RequestManager.java

```diff
@@ -372,7 +372,7 @@ void onRequestSent(long correlationId, long timeMs) {
         @Override
         public String toString() {
             return String.format(
-                "ConnectionState(node=%s, state=%s, lastSendTimeMs=%d, lastFailTimeMs=%d, inFlightCorrelationId=%d)",
+                "ConnectionState(node=%s, state=%s, lastSendTimeMs=%d, lastFailTimeMs=%d, inFlightCorrelationId=%s)",
```

Comment:
`inFlightCorrelationId` is `OptionalLong`, so its `toString` includes the prefix "OptionalLong". Could you please do a bit refactor to remove that?

Reply:
Of course, it has been modified.